src/PointwiseFunctions/AnalyticData/CurvedWaveEquation/RegularSphericalWave3DKerrSchild.cpp

```diff
@@ -0,0 +1,21 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "PointwiseFunctions/AnalyticData/CurvedWaveEquation/RegularSphericalWave3DKerrSchild.hpp"
+
+/// \cond
+namespace CurvedScalarWave ::AnalyticData {
+template class ScalarWaveGr<ScalarWave::Solutions::RegularSphericalWave,
```

Comment:
Not a change request: Ideally we'd like to document what instantiations we have, but I don't really know of a good way to do that other than manually keeping track of it in the class template documentation. Just a note in case anyone comes up with ideas in the future :)